chromium_src/chrome/browser/ui/webui/help/brave_version_updater_mac_unittest.mm

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "base/test/scoped_feature_list.h"
+#include "brave/chromium_src/chrome/browser/ui/webui/help/version_updater_mac.h"
```

Comment:
Below doesn't work?
```suggestion
#include "chrome/browser/ui/webui/help/version_updater_mac.h"
```

Reply:
It does, thanks. Resolved in eb555387b82448408280f9ef299d288bf032d1d1.